### PR TITLE
Fixed Settings.Notification and tweaked something

### PR DESCRIPTION
**Pull request by kaiserandaxl**
# Fixed Settings.Notification
I fixed Settings.Notification before it used to not show any notification, but now it shows a notification

Before: (By: @astrohweston) That's their discord not github
Medal_aFiDDIWYJC

After: (By: me)
image

# Tweaked how the system waits before sending another default notification
Before the tweak, it used to wait after every if statement. But now, it waits after a successful if statement, so instead of waiting 2 seconds for the GUI if the first if statement failed, you're only waiting 1 seconds as I put the task.wait() inside the if statement if it was successful

[**Before**](https://github.com/Epix-Incorporated/Adonis/assets/122803145/7fecd49e-29c3-4ee8-b062-dc8c9e5e3c0e) (By: Me)

[**After**](https://github.com/Epix-Incorporated/Adonis/assets/122803145/1f6bd168-ba3b-4cd1-86f5-dd0a668410a2) (By: Me)

### CODE REVIEW

File: MainModule/Server/Core/Process.lua [hunk 1]
```diff
@@ -977,8 +999,19 @@ return function(Vargs, GetEnv)
...
-							)
+								]]);
+							})
+						end
+						
+						if level >= 300 and #Settings.Messages > 0 then
+							task.wait(1)
+							for _,Message in pairs(Settings.Messages) do
+								Remote.MakeGui(p,"Notification",{
+									Title = "Message!";
```
Comment (Dimenpsyonal) [inline MainModule/Server/Core/Process.lua:1010]: Message!

Comment (kaiserandaxl) [reply MainModule/Server/Core/Process.lua:1010]: @Dimenpsyonal What about it?

**Comment by Dimenpsyonal** [comment]
ok somethign went wrong here

**Comment by kaiserandaxl** [comment]
Who changed something?

**Comment by kaiserandaxl** [comment]
Converted everything back.

**Pull request merged**